microsoft-365/security/office-365-security/threat-explorer.md

```diff
@@ -74,9 +74,42 @@ You should also see additional details within the product. For example, the tota
 
 ![View In-product Info](../media/ProductInfo.png)
 
+## Extended capabilities in Threat Explorer
+
+### Top Targeted Users
+
+Today we expose the list of Top Targeted users in the Malware View for Emails (within the Top Malware Families section). We will be extending this view within Phish and All Email views as well, where you will be able to see the top 5 targeted Users along with no. of attempts for each user for corresponding view (e.g. for Phish view you will be able to see the number of Phish attempts ).
+You will also be able to export the list of Targeted users up to a limit of 3000 along with the no. of attempts for offline analysis for each email view. In addition to that, clicking on No. of attempts (e.g. 13 attempts below) would open a filtered view in Threat Explorer, so that you can look at more details across emails and threats for that user . 
+
+![Top Targeted Users](../../media/Top_Targeted_Users.png)
+
+
+### Exchange Transport Rules
+As part of data enrichment, you should also be able to see all the different transport rules which were applied on a message. This information will be present within the Email grid view (to view this, click on Column options in the grid add Exchange Transport Rule from the Column Options in the Grid) as well as Details flyout in the email.
+You would be able to see both the GUID as well as the name of the Transport rules which were applied on the message. Additionally, you would also be able to search for the messages using the name of the Transport rule. This would be a ‘Contains’ search which means you will be able to search using Partial searches as well. 
```

Comment:
```suggestion
You would be able to see both the GUID as well as the name of the transport rules which were applied to the message. Additionally, you would be able to search for the messages using the name of the transport rule. This would be a ‘Contains’ search which means you will be able to search using partial searches as well. 
```

Reply:
JohanFreelancer9 can you update these changes directly, since this allows edits by maintainers?